types/index.test-d.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference types="webdriverio/sync" />
+/// <reference types="webdriverio" />
```

Comment:
Whats this change for?

Reply:
in wdio v8, types are present there, so this version of our SDK will support v8 types.

similar was the case when migrating from v6 -> v7 https://github.com/percy/percy-webdriverio/pull/384